testing/tools/integration/integration.py

```diff
@@ -756,10 +756,34 @@ def ex_validate(cmd):
 def setup_resilience_path(res_dir):
     # create resilience data directory if it doesn't already exist
     if not os.path.exists(res_dir):
+        create_resilience_dir(res_dir)
+    empty_resilience_dir(res_dir)
+
+def create_resilience_dir(res_dir):
+    try:
         os.mkdir(res_dir)
+    except Exception as e:
+        print 'Warning: mkdir %s failed: %s' % (res_dir, e)
+
+def delete_resilience_dir(res_dir):
+    try:
+        os.rmdir(res_dir)
+    except Exception as e:
+        print 'Warning: rmdir %s failed: %s' % (res_dir, e)
```

Comment:
I'm not sure if this should raise an error or not, so either use `logging.exception(...)` to raise an error, or `logging.warning` to just print out your message and the exception data without reraising the error.